WordPress/src/androidTest/java/org/wordpress/android/ui/stats/RemoteTests.java

```diff
@@ -1,673 +0,0 @@
-package org.wordpress.android.ui.stats;
```

Comment:
Why was this file deleted?

Reply:
I've moved it into a separate PR. I doesn't really belong here but the code was broken because the old stats code was already removed. It's just that we never run/build the connected tests